src/pages/DomainDetailsPage.vue

```diff
@@ -0,0 +1,414 @@
+<script setup lang="ts">
+import { useRouter } from 'vue-router';
+import { computed, onMounted, ref } from 'vue';
+import * as http from '@/shared/api';
+import BaseContentBlock from '@/shared/ui/components/BaseContentBlock.vue';
+import DataField from '@/shared/ui/components/DataField.vue';
+import { useTable } from '@/shared/lib/table';
+import BaseTable from '@/shared/ui/components/BaseTable.vue';
+import BaseHash from '@/shared/ui/components/BaseHash.vue';
+import { useWindowSize } from '@vueuse/core';
+import BaseLoading from '@/shared/ui/components/BaseLoading.vue';
+import { useErrorHandlers } from '@/shared/ui/composables/useErrorHandlers';
+import invariant from 'tiny-invariant';
+import type { Domain, DomainId } from '@/shared/api/schemas';
+import { getMetadata } from '@/shared/ui/utils/json';
+
+const router = useRouter();
+const { handleUnknownError } = useErrorHandlers();
+
+const HASH_BREAKPOINT = 960;
+const { width } = useWindowSize();
+
+const hashType = computed(() => (width.value < HASH_BREAKPOINT ? 'short' : 'medium'));
+
+const domainId = computed(() => {
+  const id = router.currentRoute.value.params['id'];
+
+  invariant(typeof id === 'string', 'Expected string');
+
+  return id as DomainId;
+});
+
+const domain = ref<Domain | null>(null);
+const isFetchingDomain = ref(false);
+
+const isEmptyAssets = ref(false);
+const isEmptyAccounts = ref(false);
+
+onMounted(async () => {
+  try {
+    isFetchingDomain.value = true;
+    domain.value = await http.fetchDomain(domainId.value);
+
+    if (domain.value) {
+      await Promise.all([
+        assetsTable.fetch({ domain: domainId.value }),
+        accountsTable.fetch({ domain: domainId.value }),
+      ]);
+    }
+
+    isEmptyAssets.value = !assetsTable.items.value.length;
+    isEmptyAccounts.value = !accountsTable.items.value.length;
+  } catch (e) {
+    handleUnknownError(e);
+  } finally {
+    isFetchingDomain.value = false;
+  }
+});
+
+const accountsTable = useTable(http.fetchAccounts);
+const assetsTable = useTable(http.fetchAssetDefinitions);
```

Comment:
Both endpoints support filtering by `domain`. Use it please.

Reply:
I have already used it above:
```
await Promise.all([
        assetsTable.fetch({ domain: domainId.value }),
        accountsTable.fetch({ domain: domainId.value }),
      ]);
```

Reply:
Aah, it forwards arguments. My bad.